source/1_engine/rex_engine/include/rex_engine/diagnostics/win/hr_call.h

```diff
@@ -13,7 +13,7 @@
 #pragma once
 
 #include "rex_std/bonus/string/stack_string.h"
-#include "rex_std/string_view.h"
+#include "rex_std/internal/string_view/basic_string_view.h"
```

Comment:
Can you replace the internal include to the regular one within rex please?
This comment applies to all the includes within rex